cluster/charts/rook-ceph/values.yaml.tmpl

```diff
@@ -47,6 +47,10 @@ rbacEnable: true
 ##
 pspEnable: true
 
+## If true, prefix all cluster resources to allow multiple operators run concurrently
+##
+multipleOperators: true
```

Comment:
What if this property were called: `resourcePrefix`? The default would be blank. Or it could be set to different values to support multiple operators. This way we're not forcing a certain prefix when there are multiple operators. 

A variable such as `{{.Release.Name}}` could be used for the prefix, or does helm allow variables to be used here?

Reply:
afaik it's not supported, i would like to see it though...